### PR TITLE
feat: Implement add node drawer - EXO-63070 - Meeds-io/meeds#51

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
this change is to implement the add node drawer

### CODE REVIEW

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties [hunk 1]
```diff
@@ -18,3 +18,15 @@ siteNavigation.label.btn.save=Save
 siteNavigation.label.btn.cancel=Cancel
 siteNavigation.label.moveUp=Move up
 siteNavigation.label.moveDown=Move down
+siteNavigation.label.addNode.title=Add node
+siteNavigation.label.nodeLabel.title=Node Label
```
Comment (azayati) [inline layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties]:
```suggestion
siteNavigation.label.nodeLabel.title=Node label
```

File: layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties [hunk 2]
```diff
@@ -18,3 +18,15 @@ siteNavigation.label.btn.save=Save
 siteNavigation.label.btn.cancel=Cancel
 siteNavigation.label.moveUp=Move up
 siteNavigation.label.moveDown=Move down
+siteNavigation.label.addNode.title=Add node
```
Comment (azayati) [inline layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties]:
```suggestion
siteNavigation.drawer.addNode.title=Add node
```

File: layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationAddNodeDrawer.vue [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+</template>
+<script>
+export default {
+  data: () => ({
+    visible: true,
+    scheduleVisibility: false,
+    nodeType: 'Group',
+    rules: [
+      value => {
+        return !(value && value.length) || 'Field is required.';
```
Comment (azayati) [inline layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationAddNodeDrawer.vue]: 'Field is required' should be internationalized

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**